include/kframework/lemmas/bytes-simplification.k

```diff
@@ -0,0 +1,108 @@
+module BYTES-SIMPLIFICATION
+    imports BYTES-SIMPLIFICATION-HASKELL
+endmodule
+
+module BYTES-SIMPLIFICATION-HASKELL [symbolic, kore]
+    imports BYTES-SIMPLIFICATION-COMMON
+    imports INT-SYMBOLIC
+    imports EVM
+    imports BUF
+
+  // ########################
+  // Buffer Reasoning
+  // ########################
+
```

Comment:
We can just combine the modules into one `BYTES-SIMPLIFICATION [symbolic]` instead of having `COMMON` and `HASKELL` module now.

Reply:
Makes sense, I've made that change.